docs/guide/snapshot.md

```diff
@@ -237,5 +237,32 @@ exports[`toThrowErrorMatchingSnapshot: hint 1`] = `"error"`;
 
 In Vitest, the equivalent snapshot will be:
 ```console
-exports[`toThrowErrorMatchingSnapshot > hint 1`] = `"error"`;
+exports[`toThrowErrorMatchingSnapshot > hint 1`] = `[Error: error]`;
+```
+
+#### 4. default `Error` snapshot is different for `toThrowErrorMatchingSnapshot` and `toThrowErrorMatchingInlineSnapshot`
+
+```js
+test('snapshot', () => {
+  //
+  // in Jest
+  //
+
+  expect(new Error('error')).toMatchInlineSnapshot(`[Error: error]`)
+
+  // Jest snapshots `Error.message` for `Error` instance
+  expect(() => {
+    throw new Error('error')
+  }).toThrowErrorMatchingInlineSnapshot(`"error"`)
```

Comment:
I verified this behavior in https://stackblitz.com/edit/jest-example-t1qnq1?file=sum.test.js

<details><summary>reveal screenshot</summary>

![image](https://github.com/vitest-dev/vitest/assets/4232207/0cd51d10-73ae-4e0a-8207-e5367ae7b7e8)


</details>